components/dependency_injection/advanced.rst

```diff
@@ -219,6 +219,79 @@ You can change the inner service name if you want to:
             ->setPublic(false)
             ->setDecoratedService('foo', 'bar.wooz');
 
+.. versionadded:: 2.8
+    The ability to define the decoration priority was introduced in Symfony 2.8.
+    Prior to Symfony 2.8, the priority depends on the order in
+    which definitions are found.
+
+If you want to apply more than one decorator to a service, you can control their
+order by configuring the priority of decoration. Decorators with higher
```

Comment:
Maybe:

> If you want to apply more than one decorator to a service, you can control their
order by configuring the priority of decoration, this can be any integer number (decorators with higher priorities will be applied first)

What do you think? I'm not sure if we should say "integer numbers" or "positive and negative numbers". I prefer the first one because it's short and mathematically correct.